mlrun/api/utils/projects/member.py

```diff
@@ -147,3 +149,36 @@ def get_project_owner(
         name: str,
     ) -> mlrun.api.schemas.ProjectOwner:
         pass
+
+    async def on_after_delete_project(
+        self,
+        project_name: str,
+    ):
+        logger.debug(
+            "Running on_after_delete_project",
+            project=project_name,
+        )
+        await self._stop_logs_for_project(project_name)
+
+    @staticmethod
```

Comment:
shouldn't be a staticmethod even if it's not using any self attributes since it's private

Reply:
is this an mlrun convention?

Reply:
discussed offline